src/main/java/Domain/Permission.java

```diff
@@ -0,0 +1,49 @@
+package Domain;
+/**
+ * This enum represents the permissions that a user with a role in a specific shop has. 
+ */
+public enum Permission {
+
+    //CAN DO ALL
+    FOUNDER,
+
+    //FOUNDER ACTIONS //TODO: Maybe not needed and we can use just FOUNDER.
```

Comment:
good point, lets talk about that tonigth

Reply:
ok